models/engine/db_storage.py

```diff
@@ -74,3 +74,20 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """returns object with key matching cls and id"""
+        if cls is not None and id is not None:
+            test = self.all(cls)
+            word = "State.{}".format(id)
```

Comment:
I'm not sure what word does in this code. Is there an easier way to refactor this to make it readable? Maybe writing it in a for loop like what you wrote, but instead you can write for values in self.all(class).values() ---> and the check, if values.id == id ---> return values.